### PR TITLE
[Cognitive Services] Bump cognitiveservices to 2021-10-01

### PR DESCRIPTION
**Pull request by yangyuan**
**Description**
Cognitive Services CLI upgrade to API 2021-10-01

**Testing Guide**
`cognitiveservices account deployment [show,list,create,delete]`
`cognitiveservices account commitment-plan [show,list,create,delete]`
`cognitiveservices commitment-tier [list]`

**History Notes**

[Cognitive Services] add new commands `cognitiveservices account deployment [show,list,create,delete]`
[Cognitive Services] add new commands `cognitiveservices account commitment-plan [show,list,create,delete]`
[Cognitive Services] add new commands `cognitiveservices commitment-tier [list]`

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yangyuan** [comment]
I think something is wrong with CI.
It says 
```
-  FAIL - HIGH severity: missing_group_help
    Command-Group: `cognitiveservices account commitment-plan` - Missing help
    Command-Group: `cognitiveservices account deployment` - Missing help
    Command-Group: `cognitiveservices account identity` - Missing help
    Command-Group: `cognitiveservices account keys` - Missing help
    Command-Group: `cognitiveservices account network-rule` - Missing help
    Command-Group: `cognitiveservices account` - Missing help
    Command-Group: `cognitiveservices commitment-tier` - Missing help
    Command-Group: `cognitiveservices` - Missing help
```
But I do have group helps such as 
https://github.com/Azure/azure-cli/blob/3d3930651d4e23602285ca2a5c4878aa34c1c374/src/azure-cli/azure/cli/command_modules/cognitiveservices/_help.py#L16

**Comment by yonzhan** [comment]
Cognitive Services

**Comment by yangyuan** [comment]
looks like its failing with the same error.

File: src/azure-cli/azure/cli/command_modules/cognitiveservices/__init__.py [hunk 1]
```diff
@@ -5,19 +5,14 @@
 
 from azure.cli.core import AzCommandsLoader
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/cognitiveservices/__init__.py:7]:
```suggestion

import azure.cli.command_modules.cognitiveservices._help # pylint: disable=unused-import
```

Comment (kairu-ms) [reply src/azure-cli/azure/cli/command_modules/cognitiveservices/__init__.py:7]: @yangyuan It's where the issue comes from. You should not remove it.

File: src/azure-cli/azure/cli/command_modules/cognitiveservices/__init__.py [hunk 2]
```diff
@@ -5,19 +5,14 @@
...
-
 
 class CognitiveServicesCommandsLoader(AzCommandsLoader):
 
     def __init__(self, cli_ctx=None):
         from azure.cli.core.commands import CliCommandType
         from azure.cli.core.profiles import ResourceType
         custom_type = CliCommandType(
-            operations_tmpl='azure.cli.command_modules.cognitiveservices.custom#{}',
-            client_factory=cf_accounts)
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/cognitiveservices/__init__.py:20]: Please keep these unchanged.

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**